converter/internal/prometheusconvert/prometheusconvert_test.go

```diff
@@ -36,17 +34,13 @@ func TestConvert(t *testing.T) {
 
 			t.Run(caseName, func(t *testing.T) {
 				actual, diags := prometheusconvert.Convert(inputBytes)
+				os.WriteFile("/home/erik/temp/"+caseName+".river", actual, 0666)
```

Comment:
this looks pretty sus ;) Also we may want this to be linux only?

Reply:
Haha woops. Tests work in both windows/linux after normalizing the line endings